tools/make-one-time-file.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import sys
+from TimeFileMaker import *
```

Comment:
I'm not expert in python.  Can you clarify how TimeFileMaker.py is found? (it is installed in COQLIB as far as I can see)